tests/code_layout/doxygen_parser.py

```diff
@@ -271,9 +271,14 @@ def parseClassElem(self, e):
             for s in para.iter('simplesect'):
                 if s.get('kind') == 'since':
                     for p in s.iter('para'):
-                        if self.version_regex.match(p.text):
+                        if self.version_regex.match(ET.tostring(p).decode()):
                             found_version_added = True
                             break
+            for s in para.iter('xrefsect'):
+                if s.find('xreftitle') is not None and 'Deprecated' in s.find('xreftitle').text:
+                    # can't have both deprecated and since, so if we've found deprecated then treat it as having satisified the "since" requirement too
```

Comment:
```suggestion
                    # can't have both deprecated and since, so if we've found deprecated then treat it as having satisfied the "since" requirement too
```